pkg/volume/glusterfs/glusterfs.go

```diff
@@ -467,7 +467,7 @@ func (p *glusterfsVolumeProvisioner) CreateVolume() (r *api.GlusterfsVolumeSourc
 		glog.Errorf("glusterfs: failed to create gluster rest client")
 		return nil, 0, fmt.Errorf("failed to create gluster REST client, REST server authentication failed")
 	}
-	volumeReq := &gapi.VolumeCreateRequest{Size: sz}
+	volumeReq := &gapi.VolumeCreateRequest{Size: sz, Durability: gapi.VolumeDurabilityInfo{Type:gapi.DurabilityReplicate, Replicate:gapi.ReplicaDurability{Replica:3}}}
```

Comment:
@rootfs yes, these are the current defaults. Its possible to parameterize in storage class, however it may not add value now as we recommend/configure distribute-replicate(3) volumes by default. I will check this once again with the team and confirm though